backend/app/views/spree/admin/promotion_code_batches/download.csv.ruby

```diff
@@ -0,0 +1,6 @@
+CSV.generate do |csv|
+  csv << ['Code']
```

Comment:
I don't know if I agree with CSV headers being i18n'd, as they might need to be parsed by external systems... so I'm also a toss up. So I'm probably good with this as a default, and as Thomas says letting people override if they don't like.